src/buddy/sign/jws.clj

```diff
@@ -147,5 +147,14 @@
                        {:type :validation :cause :signature})))
      (decode-payload payload))))
 
+(defn decode-no-signature
```

Comment:
Maybe this function should be named something more dire like: `unsafe-decode-without-checking-signature`?, and a docstring should be added to explain what this is doing, and that it shouldn't be used in production code.